source/_integrations/media_player.markdown

```diff
@@ -178,3 +178,64 @@ The way media players are displayed in the frontend can be modified in the [cust
 - `tv`: Device is a television type device.
 - `speaker`: Device is speaker or stereo type device.
 - `receiver`: Device is audio video receiver type device taking audio and outputting to speakers and video to some display.
+
+## Example script
+
+The example script below combines a number of media player commands to set the output speaker for [Spotify](https://www.home-assistant.io/integrations/spotify) to 30% volume and plays a randomly shuffled track from the playlist Calming Classical.
+
+This demonstrates how steps a user would go through manually can be combined into a script that can then be called by a single button press or automation.
+
+```yaml
+# configuration.yaml
+script:
+  demo_spotify:
```

Comment:
Please make sure this example complies with our YAML documentation style guide.

https://developers.home-assistant.io/docs/documenting/yaml-style-guide